katdal/datasources.py

```diff
@@ -623,42 +622,49 @@ def from_url(cls, url, chunk_store='auto', upgrade_flags=True, **kwargs):
         kwargs : dict, optional
             Extra keyword arguments passed to telstate view and chunk store init
         """
-        url_parts = urllib.parse.urlparse(url, scheme='file')
+        u = urllib.parse.urlparse(url, scheme='file')
```

Comment:
Any particular reason for making this variable name less self-documenting?

Reply:
Reducing line lengths to fit into my editor window.

Reply:
Reverted.

Reply:
Yeah, I think meaningful variable names are more important than not have to scroll.